helm/mds/templates/configmap.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
   name: mds-config-files
   namespace: {{ .Release.Namespace }}
 data:
-{{ (.Files.Glob "files/mds-config/*.json").AsConfig | indent 2 }}
-{{ (.Files.Glob "files/mds-config/*.json5").AsConfig | indent 2 }}
+{{ (.Files.Glob "files/organization-config/*.json").AsConfig | indent 2 }}
+{{ (.Files.Glob "files/organization-config/*.json5").AsConfig | indent 2 }}
```

Comment:
How important is it that these match? I have a minor concern that the concept of an `organization` is not really defined in this context. Maybe `config-settings`?